tests/e2e/test_e2e.py

```diff
@@ -549,6 +566,9 @@ def regression_bounded(model, test_fraction=0.02):
         classification_binary(
             ensemble.RandomForestClassifier(**FOREST_PARAMS)),
     ],
+    [
+        (R, REGRESSION_WITH_MISSING_VALUES),
```

Comment:
Code, generated for R fails on Nan input with
````
Error in if (NaN <= 1) { : missing value where TRUE/FALSE needed
````
skipping for now

Reply:
@StrikerRUS quick research didn't expose an obvious way to work this around. Neither `NA` nor `NaN` behave in the expected way in comparison expressions. Is there any chance you're aware of some workaround that can help us out here?

Reply:
Indeed, in R you got `NA` when compare `NaN`s
```
print(NaN <= 1)  # [1] NA
```

It is possible to wrap condition into `isTRUE()` function, but I'm not sure whether it is the right way:
```
print(isTRUE(NaN <= 1))  # [1] FALSE
```